### PR TITLE
Mini Challenge Pull Request by Thommzy[Obeisun Timothy]

### PR DESCRIPTION
**Pull request by Thommzy**
##### By Thommzy

### CODE REVIEW

**Comment by bondz** [comment]
Sorry but you need to update your branch with the latest changes from master
```bash
git pull upstream master
```

File: src/roman-numerals/index.js [hunk 1]
```diff
@@ -4,6 +4,35 @@
...
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {
+  let final = 0;
+  let cur;
+  let curVal;
+  let next;
+  let nextVal;
+  const newObject = new Object();
```
Comment (bondz) [inline src/roman-numerals/index.js:13]: Why not just use a regular object here.

Comment (Thommzy) [reply src/roman-numerals/index.js:13]: Okay.. will use a regular Object next time.
Thank you.

**Comment by Thommzy** [comment]
> Your variable names need to be more descriptive.

Okay. I will work on that .. thank you.

**Pull request merged**